{"name": "test_table_pagination", "status": "broken", "statusDetails": {"message": "TypeError: 'int' object is not callable", "trace": "self = <Tests.test_table_pagination.TestTablePagination object at 0x000001D291444AC0>\n\n    def test_table_pagination(self):\n        self.logger.info(\"*************** Test_001_Table_Pagination *****************\")\n    \n        self.logger.info(\"*************** Test Table Pagination Started *****************\")\n    \n        self.obj.open(TablePaginationLocators.TablePaginationUrl)\n        time.sleep(3)\n        # test form\n        total_rows = self.obj.check_records_total_num()\n        page1_rows = self.obj.check_page1_rows_number()\n        page2_rows = self.obj.check_page2_rows_number()\n        page3_rows = self.obj.check_page3_rows_number()\n    \n>       self.logger.level(logging.DEBUG)\nE       TypeError: 'int' object is not callable\n\nTests\\test_table_pagination.py:32: TypeError"}, "attachments": [{"name": "log", "source": "ece49b18-51a6-462f-8722-c9e8f9be427b-attachment.txt", "type": "text/plain"}], "start": 1627429803797, "stop": 1627429809953, "uuid": "1d757fa9-bcdc-458e-8e4c-0d6c746d5392", "historyId": "bc31245b68d8b1043faea669f381b3b5", "testCaseId": "279e5997153c778fda4d099070fbb23c", "fullName": "Tests.test_table_pagination.TestTablePagination#test_table_pagination", "labels": [{"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_table_pagination"}, {"name": "subSuite", "value": "TestTablePagination"}, {"name": "host", "value": "DESKTOP-K3935TM"}, {"name": "thread", "value": "16168-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_table_pagination"}]}